src/utils.js

```diff
@@ -1477,6 +1477,9 @@ const bundleOrders = (
     for (let i = 0; i < ordersDetails.length; i++) {
         const orderDetails = ordersDetails[i];
         const orderStruct = JSON.parse(ordersDetails[i].orderJSONString);
+        // exchange the "handleIo" to "handleIO"
+        orderStruct.handleIO = orderStruct.handleIo;
```

Comment:
if `handleIo` is NOT set and the bug is fixed upstream then this will cause `handleIO` to be broken